server/queue.lua

```diff
@@ -18,7 +18,7 @@ end
 -- Queue code
 
 local config = require 'config.queue'
-local maxPlayers = GlobalState.MaxPlayers
+local maxPlayers = GetConvarInt('sv_maxclients', 48)
```

Comment:
```suggestion
local maxPlayers = GlobalState.MaxPlayers or GetConvarInt('sv_maxclients', 48)
```
Might be slightly more bulletproof?

Reply:
I don't see a reason for that? `GetConvarInt` will be fully reliable.

Reply:
if its set and gotten the same way then how is it less reliable?

Reply:
I'm guessing it's a sort of race condition? Not entirely sure why it happens but there's no reason for `GetConvarInt` to fail. I'm guessing using `GlobalState` in this case in general isn't  the most reliable.